### PR TITLE
Net analyzer/nagstamon

### PR DESCRIPTION
**Pull request by zen2**
bug #718424
    Add support for python 3.7 and 3.8
    Patch to handle optional VERSION_ID in /etc/os-release

### CODE REVIEW

File: net-analyzer/nagstamon/nagstamon-3.4.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+src_prepare() {
+	default_src_prepare
+
+	# pre-compressed already
+	rm Nagstamon/resources/nagstamon.1.gz || die
+	sed -i -e 's:\(nagstamon\.1\)\.gz:\1:' setup.py || die
+
+	mv ${PN}.py ${PN} || die
+
+	rm -rf "${S}/Nagstamon/thirdparty/Xlib/" || die
```
Comment (hydrapolic) [inline net-analyzer/nagstamon/nagstamon-3.4.1-r1.ebuild:53]: How about: rm -r "Nagstamon/thirdparty/Xlib" || die

File: net-analyzer/nagstamon/nagstamon-3.4.1-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python3_7 python3_8 )
```
Comment (hydrapolic) [inline net-analyzer/nagstamon/nagstamon-3.4.1-r1.ebuild:6]: PYTHON_COMPAT=( python3_{7,8} )

Comment (zen2) [reply net-analyzer/nagstamon/nagstamon-3.4.1-r1.ebuild]: Done.
New commit: https://github.com/gentoo/gentoo/pull/17373/commits/226d9fadd4804c27674d0ac462e1ed4a6eb8aa6f

File: net-analyzer/nagstamon/nagstamon-3.4.1-r1.ebuild (hunk 1 above)
Comment (zen2) [reply net-analyzer/nagstamon/nagstamon-3.4.1-r1.ebuild]: I didn't touch original ebuild but not bad to fix this too.
New commit: https://github.com/gentoo/gentoo/pull/17373/commits/226d9fadd4804c27674d0ac462e1ed4a6eb8aa6f

**Comment by hydrapolic** [comment]
Please squash the commits to a single one and then force push to update this branch.

**Comment by hydrapolic** [comment]
Please also add Signed-off-by into your git commit (https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin)

**Comment by arkamar** [comment]
This PR can be probably closed, the issue 718424 was fixed couple of days after this PR was submitted, see https://bugs.gentoo.org/718424#c16.

**Comment by ajakk** [comment]
Closing as obsolete, thanks arkamar!

**Pull request closed without merging**